modules/ztp-configuring-the-cluster-for-a-disconnected-environment.adoc

```diff
@@ -43,7 +43,7 @@ data:
       mirror-by-digest-only = true
 ----
 <1> The mirror registry’s certificate used when creating the mirror registry.
-<2> The configuration for the mirror registry.
+<2> The configuration file for the mirror registry. The mirror registry configuration adds mirror information to `/etc/containers/registries.conf` in the Discovery image. The mirror information is stored in the `imageContentSources` section of the `install-config.yaml` file when passed to the installer. The Assisted Service pod also converts `registries.conf` into an `imageContentSourcesPolicy` file, and uses it with the `--icsp` flag in a number of `oc adm` commands run against the Release image.
```

Comment:
`--icsp` should be `--icsp-file`

I think that the last sentece about what the assisted service is doing with the mirror information is internal implementation detail, I'm not sure if it belongs in a customer-facing docs.
However, the impact of this implementation detail does matter, in case mirror information was provided the assisted-service (that run on the HUB cluster) will fetch some images from the mirror registry. this means that the assisted-service pod that runs on the HUB needs to be able to resolve the mirror registry.